addons/medical/XEH_postInit.sqf

```diff
@@ -2,6 +2,11 @@
 
 #include "script_component.hpp"
 
+["medical_selectionDamage", {_this call FUNC(handleSelectionDamage)}] call EFUNC(common,addEventhandler);
+["medical_fallDamage",      {_this call FUNC(handleFallDamage)}]      call EFUNC(common,addEventhandler);
+["medical_drowningDamage",  {_this call FUNC(handleDrowningDamage)}]  call EFUNC(common,addEventhandler);
+["medical_collisionDamage", {_this call FUNC(handleCollisionDamage)}] call EFUNC(common,addEventhandler);
```

Comment:
Doing this might look good, but is actually bad practice to do with git in mind. Adding or changing one line and having to adjust other lines makes it so the chance of having merge conflicts is increased.